prow/plugins/override/override.go

```diff
@@ -326,6 +326,11 @@ func handle(oc overrideClient, log *logrus.Entry, e *github.GenericCommentEvent,
 			resp := "/override requires failed status contexts to operate on, but none was given"
 			log.Debug(resp)
 			return oc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, user, resp))
+		} else if regexp.MustCompile(`^\"(.+)\"$`).MatchString(m[2]) {
```

Comment:
introducing another inlined regex doesn't seem like the best fix to me. In my imagination, this could be done by  replacing line 335 with a string parsing logic, which understands how to find substrings that are either space separated, and taking into consideration of quotes

Reply:
I moved the logic to its own function